

<!DOCTYPE html>

<html lang="en" prefix="og: http://ogp.me/ns#">
    <head>
        <title>Publications</title>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, inital-scale=1" />
        

        <meta name="twitter:card" content="summary" />
        <meta name="twitter:title" content="ISOLDE: Bringing structural biology to life" />
        <meta name="twitter:description" content="Fast and flexible interactive molecular modelling into experimental maps" />
        <meta name="twitter:image" content=" /static/isolde/images/logo_square.jpg" />

        <meta property="og:title" content="ISOLDE: Bringing structural biology to life" />
        <meta property="og:image" content="/static/isolde/images/logo_square.jpg" />
        <meta property="og:description" content="Fast and flexible interactive molecular modelling into experimental maps" />
        <meta property="og:url" content="https://tristanic.github.io/isolde/publications/" />

        
        <link href="../static/isolde/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
        <link href="../static/isolde/assets/css/main.css" rel="stylesheet" type="text/css" />

        

        
    </head>

    
	<body class="subpage">

        

        <header id="header">
            <div class="logo"><a href="../index.html">ISOLDE <span>home</span></a></div>
            <a href="#menu">Menu</a>
        </header>

        <!-- Nav -->
        <nav id="menu">
            <ul class="links">
                


<li class="child ancestor">
	<a href="../index.html">Home</a>
	
	<ul>
		


<li class="child sibling">
	<a href="../what-isolde/index.html">What is ISOLDE?</a>
	
</li>

<li class="child sibling">
	<a href="../download/index.html">Download</a>
	
</li>

<li class="child sibling">
	<a href="../features-main/index.html">Features</a>
	
	<ul>
		


<li class="child">
	<a href="../features-main/features-graphics/index.html">Graphics</a>
	
</li>

<li class="child">
	<a href="../features-main/features-interactive-md/index.html">Interactive MD</a>
	
</li>

<li class="child">
	<a href="../features-main/real-time-validation/index.html">Real-time Validation</a>
	
</li>

<li class="child">
	<a href="../features-main/live-atomic-symmetry/index.html">Live atomic symmetry</a>
	
</li>

<li class="child">
	<a href="../features-main/interactive-restraints/index.html">Interactive restraints</a>
	
</li>

<li class="child">
	<a href="../features-main/real-time-crystallographic-map-calculations/index.html">Real-time crystallographic map calculations</a>
	
</li>

	</ul>
	
</li>

<li class="child sibling">
	<a href="../faq/index.html">Frequently Asked Questions</a>
	
</li>

<li class="child sibling">
	<a href="../documentation/index.html">Documentation</a>
	
</li>

<li class="child sibling">
	<a href="../performance-benchmarks/index.html">Performance</a>
	
</li>

<li class="child sibling">
	<a href="../about/index.html">About</a>
	
</li>

<li class="child sibling">
	<a href="../licensing/index.html">Licensing</a>
	
</li>

<li class="child selected">
	<a href="index.html">Publications</a>
	
</li>

<li class="child sibling">
	<a href="../case-studies/index.html">Case Studies</a>
	
	<ul>
		


<li class="child">
	<a href="../case-studies/case-study-3io0/index.html">3io0</a>
	
</li>

<li class="child">
	<a href="../case-studies/case-study-3loh/index.html">3loh</a>
	
</li>

<li class="child">
	<a href="../case-studies/case-study-3j9d/index.html">3j9d</a>
	
</li>

	</ul>
	
</li>

<li class="child sibling">
	<a href="../contact/index.html">Contact</a>
	
</li>

	</ul>
	
</li>

            </ul>
        </nav>


        
		<!-- One -->
			<section id="One" class="wrapper style3">
				<div class="inner">
					<header class="align-center">
						<p>ISOLDE: Bringing structural biology to life</p>
						<h2>Publications</h2>
					</header>
				</div>
			</section>

		<!-- Two -->
			<section id="two" class="wrapper style2">
				<div class="inner">
					<div class="box">
						<div class="content">
                            
    							<header class="align-center">
    								
    							</header>
    							

<div class="container">
    
        

<div class="row">
    
        

<div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">
    
        

<div class="list-group">
    
        <a href="index.html#publications" class>Peer-reviewed papers</a>
    
        <a href="index.html#presentations" class="btn btn-link">Posters, presentations and workshops</a>
    
        <a href="index.html#structures" class="btn btn-link">Published structures using ISOLDE</a>
    
</div>

    
        <h2>Citing ISOLDE</h2>

<p>If you use ISOLDE in work leading to a publication, please read and cite the following paper:</p>

<p>Croll, Tristan Ian. 2018. “ISOLDE: A Physically Realistic Environment for Model Building into Low-Resolution Electron-Density Maps.” <i>Acta Crystallographica Section D: Structural Biology </i>74 (6):519–30. doi: <a href="https://doi.org/10.1107/S2059798318002425" target="_blank">10.1107/S2059798318002425</a>.</p>
    
</div>

    
</div>

    
</div>


<div id="publications" class="container">
    
        

<div class="row">
    
        

<div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">
    
        <h2>Publications related to ISOLDE or IMDFF</h2>

<h3>2021</h3>

<p>Croll, Tristan I and Read, R. 2021. "Adaptive Cartesian and torsional restraints for interactive model rebuilding." <em>Acta Crystallographica Section D: Biological Crystallography. </em>In press.</p>

<p>Thomson, Emma C, Laura E Rosen, James G Shepherd, Roberto Spreafico, Ana da Silva Filipe, Jason A Wojcechowskyj, Chris Davis, Luca Piccoli, David J Pascall, Josh Dillen, Spyros Lytras, Nadine Czudnochowski, Rajiv Shah, Marcel Meury, Natasha Jesudason, Anna De Marco, Kathy Li, Jessica Bassi, Aine O’Toole, Dora Pinto, Rachel M Colquhoun, Katja Culap, Ben Jackson, Fabrizia Zatta, Andrew Rambaut, Stefano Jaconi, Vattipally B Sreenu, Jay Nix, Ivy Zhang, Ruth F Jarrett, William G Glass, Martina Beltramello, Kyriaki Nomikou, Matteo Pizzuto, Lily Tong, Elisabetta Cameroni, Tristan I Croll, Natasha Johnson, Julia Di Iulio, Arthur Wickenhagen, Alessandro Ceschi, Aoife M Harbison, Daniel Mair, Paolo Ferrari, Katherine Smollett, Federica Sallusto, Stephen Carmichael, Christian Garzoni, Jenna Nichols, Massimo Galli, Joseph Hughes, Agostino Riva, Antonia Ho, Marco Schiuma, Malcolm G Semple, Peter JM Openshaw, Elisa Fadda, J Kenneth Baillie, John D Chodera, Suzannah J Rihn, Samantha J Lycett, Herbert W Virgin, Amalio Telenti, Davide Corti, David L Robertson, Gyorgy Snell, and ISARIC4C Investigators. 2021. "Circulating SARS-CoV-2 spike N439K variants maintain fitness while evading antibody-mediated immunity." <em>Cell</em> 184(5): 1171-1187.e20. doi: <a href="https://doi.org/10.1016/j.cell.2021.01.037">10.1016/j.cell.2021.01.037</a></p>

<p>Pettersen, Eric F, Thomas D Goddard, Conrad C Huang, Elaine C Meng, Gregory S Couch, Tristan I croll, John H Morris, and Thomas E Ferrin. 2021. "UCSF ChimeraX: Structure visualization for researchers, educators, and developers." <em>Protein Science</em> 30(1): 70-82. doi: <a href="https://doi.org/10.1002/pro.3943">10.1002/pro.3943</a></p>

<p>Croll, Tristan I, Christopher J Williams, Vincent B Chen, David C Richardson, and Jane S Richardson. 2021. "Improving SARS-CoV-2 structures: peer review by early coordinate release." <em>Biophysical Journal</em> online ahead of print. doi: <a href="https://doi.org/10.1016/j.bpj.2020.12.029">10.1016/j.bpj.2020.12.029</a></p>

<h3>2020</h3>

<p>Retana, Tania R., Jack Bradley-Clarke, Tristan Croll, Ruth Rose, Inva Hoti, Andrew J Stagg, Mayte Villalba, and Richard W. Pickersgill. 2020. "Lig v 1 structure and the inflammatory response to the Ole e 1 protein family." <em>Allergy </em>75(9): 2395-2398. doi: <a href="https://doi.org/10.1111/all.14351">10.1111/all.14351</a></p>

<p>Lonhienne, Thierry, Yu Shang Low, Mario D. Garcia, Tristan Croll, Yan Gao, Quan Wang, Lou Brillault, Craig M. Williams, James A. Fraser, Ross P. McGeary, Nicholas P. West, Michael J. Landsberg, Zihe Rao, Gerhard Schenk, and Luke W. Guddat. 2020. "Structures of fungal and plant acetohydroxyacid synthases." <em>Nature </em>586(7828): 317-321. doi: <a href="https://doi.org/10.1038/s41586-020-2514-3">10.1038/s41586-020-2514-3</a></p>

<p>Xu, Yibin, Nicholas S. Kirk, Hariprasad Venugopal, Mai B. Margetts, Tristan I. Croll, Jarrod J. Sandow, Andrew I. Webb, Carlie A. Delaine, Briony E. Forbes, and Michael C. Lawrence. 2020. "How IGF-II binds to the human type 1 insulin-like growth factor receptor." <em>Structure</em> 28(7): 786-798.e6. doi: <a href="https://doi.org/10.1016/j.str.2020.05.002">10.1016/j.str.2020.05.002</a></p>

<p>Woo, Hyeonuk, Sang-Jun Park, Yeol Kyo Choi, Taeyong Park, Maham Tanveer, Yiwei Cao, Nathan R. Kern, Jumin Lee, Min Sun Yeom, Tristan I. Croll, Chaok Seok, and Wonpil Im. 2020. "Developing a fully-glycosylated full-length SARS-CoV-2 spike protein model in a viral membrane." <em>Journal of Physical Chemistry B </em>124(33): 7128-7137. doi: <span class="identifier doi"><a class="id-link" data-ga-action="DOI" data-ga-category="full_text" href="https://doi.org/10.1021/acs.jpcb.0c04553" target="_blank">0.1021/acs.jpcb.0c04553 </a></span></p>

<p>Rougé, Lionel, Nancy Chiang, Micah Steffek, Christine Kugel, Tristan I. Croll, Christine Tam, Alberto Estevez, Christopher P. Arthur, Christopher M. Koth, Claudio Ciferri, Edward Kraft, Jian Payandeh, Gerald Nakamura, James T. Koerber2, and Alexis Rohou. 2020. "Structure of CD20 in complex with the therapeutic monoclonal antibody rituximab." <em>Science</em> 367(6483): 1224-1230. doi:  <a href="https://doi.org/10.1126/science.aaz9356">10.1126/science.aaz9356</a></p>

<h3>2019</h3>

<p>Liebschner, Dorothee, Pavel V Afonine, Matthew L Baker, Gabor Bunkoczi, Vincent B Chen, Tristan Croll, Bradley Hintze, Li-Wei Hung, Swati Jain, Airlie McCoy, Nigel W Moriarty, Robert D Oeffner, Billy K Poon, Michael G Prisant, Randy Read, Jane S Richardson, David C Richardson, Massimo Sammito, Oleg V Sobolev, Duncan H Stockwell, Thomas C Terwilliger, Alexandre G Urzhumtsev, Lizbeth L Videau, Christopher J Williams, and Paul D Adams. 2019. "Macromolecular structure determination using X-rays, neutrons and electrons: Recent developments in Phenix." <em>Acta Crystallographica</em> <em>D</em>75: 861-877. doi: <a href="https://doi.org/10.1107/S2059798319011471">10.1107/S2059798319011471</a></p>

<p>Piper, Sarah J., Lou Brillault, Rosalba Rothnagel, Tristan I Croll, Joseph K Box, Irene Chassagnon, Sebastian Scherer, Kenneth N Goldie, Sandra A Jones, Femke Schepers, Lauren Hartley-Tassell, Thomas Ve, Jason N Busby, Julie E Dalziel, J Shaun Lott, Ben Hankamer, Henning Stahlberg, Mark RH Hurst, and Michael J Landsberg. 2019. "Cryo-EM structures of the pore-forming A subunit from the Yersinia entomophaga ABC toxin." <em>Nature Communications </em>10 (1): 1952. doi: <a href="https://doi.org/10.1038/s41467-019-09890-8">10.1038/s41467-019-09890-8</a></p>

<p>Gao, Chen, Mitchell A Pallett, Tristan I Croll, Geoffrey L Smith, and Stephen C Graham. 2019. "Molecular basis of cullin-3 (Cul3) ubiquitin ligase subversion by vaccinia virus protein A55." <em>Journal of Biological Chemistry </em>294 (16): 6416-6429 doi: <a href="https://doi.org/10.1074/jbc.RA118.006561">10.1074/jbc.RA118.006561</a></p>

<h3>2018</h3>

<p>Croll, Tristan Ian. 2018. “ISOLDE: A Physically Realistic Environment for Model Building into Low-Resolution Electron-Density Maps.” <i>Acta Crystallographica Section D: Structural Biology </i>74 (6):519–30. doi: <a href="https://doi.org/10.1107/S2059798318002425" target="_blank">10.1107/S2059798318002425</a>.</p>

<p>McNicholas, Stuart, Tristan Croll, Tom Burnley, Colin M. Palmer, Soon Wen Hoh, Huw T. Jenkins, Eleanor Dodson, Kevin Cowtan, and Jon Agirre. 2018. “Automating Tasks in Protein Structure Determination with the Clipper Python Module.” <i>Protein Science </i>27 (1):207–16. doi: <a href="https://doi.org/10.1002/pro.3299" target="_blank">10.1002/pro.3299</a>.</p>

<p>Keough, Dianne T., Dominik Rejman, Radek Pohl, Eva Zborníková, Dana Hocková, Tristan Croll, Michael D Edstein, Geoff W Birrell, Marina Chavchich, Lieve MJ Naesens, Gregory K Pierens, Ian M Brereton, and Luke W Guddat. 2018. “Design of Plasmodium Vivax Hypoxanthine-Guanine Phosphoribosyltransferase Inhibitors as Potential Antimalarial Therapeutics.” <i>ACS Chemical Biology </i>13 (1):82–90. doi: <a href="https://doi.org/10.1021/acschembio.7b00916" target="_blank">acschembio.7b00916</a>.</p>

<h3>2017</h3>

<p>Ve, Thomas, Parimala R Vajjhala, Andrew Hedger, Tristan Croll, Frank DiMaio, Shane Horsefield, Xiong Yu, Peter Lavrencic, Zahid Hassan, Garry P Morgan, Ashley Mansell, Mehdi Mobli, Ailis O'Carroll, Brieuc Chauvin, Yann Gambin, Emma Sierecki, Michael J Landsberg, Katryn J Stacey, Edward H Egelman, and Bostjan Kobe. 2017. “Structural Basis of TIR-Domain-Assembly Formation in MAL- and MyD88-Dependent TLR4 Signaling.” <i>Nature Structural &amp; Molecular Biology </i>24 (9). doi: <a href="https://doi.org/10.1038/nsmb.3444" target="_blank">10.1038/nsmb.3444</a>.</p>

<p>Focht, Dorota, Tristan I. Croll, Bjorn P. Pedersen, and Poul Nissen. 2017. “Improved Model of Proton Pump Crystal Structure Obtained by Interactive Molecular Dynamics Flexible Fitting Expands the Mechanistic Model for Proton Translocation in P-Type ATPases.” <i>Frontiers in Physiology </i>8 (APR):202. doi: <a href="https://doi.org/10.3389/fphys.2017.00202" target="_blank">10.3389/fphys.2017.00202</a>.</p>

<p>Zhang, Xiaoxiao, Maud Bernoux, Adam R. Bentham, Toby E. Newman, Thomas Ve, Lachlan W. Casey, Tom M. Raaymakers, Jian Hu, Tristan I. Croll et al. 2017. “Multiple Functional Self-Association Interfaces in Plant TIR Domains.” <i>Proceedings of the National Academy of Sciences </i>114 (10):E2046–52. doi: <a href="https://doi.org/10.1073/pnas.1621248114" target="_blank">10.1073/pnas.1621248114</a>.</p>

<h3>2016</h3>

<p>Casey, Lachlan W, Peter Lavrencic, Adam R Bentham, Stella Cesari, Daniel J Ericsson, Tristan Croll, Dušan Turk, Peter A Anderson, Alan E Mark, Peter N Dodds, Mehdi Mobli, Bostjan Kobe, and Simon J Williams. 2016. “The CC Domain Structure from the Wheat Stem Rust Resistance Protein Sr33 Challenges Paradigms for Dimerization in Plant NLR Proteins.” <i>Proceedings of the National Academy of Sciences </i>113 (45):12856–61. doi: <a href="https://doi.org/10.1073/pnas.1609922113" target="_blank">10.1073/pnas.1609922113</a>.</p>

<p>Croll, Tristan Ian, and Gregers Rom Andersen. 2016. “Re-Evaluation of Low-Resolution Crystal Structures via Interactive Molecular-Dynamics Flexible Fitting (IMDFF): A Case Study in Complement C4.” <i>Acta Crystallographica Section D: Structural Biology </i>72 (9):1006–16. doi: <a href="https://doi.org/10.1107/S2059798316012201" target="_blank">10.1107/S2059798316012201</a>.</p>

<p>Croll, Tristan I., Brian J. Smith, Mai B. Margetts, Jonathan Whittaker, Michael A. Weiss, Colin W. Ward, and Michael C. Lawrence. 2016. “Higher-Resolution Structure of the Human Insulin Receptor Ectodomain: Multi-Modal Inclusion of the Insert Domain.” <i>Structure </i>24 (3):469–76. doi: <a href="https://doi.org/10.1016/j.str.2015.12.014" target="_blank">10.1016/j.str.2015.12.014</a>.</p>

<h3>2015</h3>

<p>Croll, Tristan Ian. 2015. “The Rate of Cis-Trans Conformation Errors Is Increasing in Low-Resolution Crystal Structures.” <i>Acta Crystallographica Section D: Biological Crystallography </i>71 (3):706–9. <a href="https://doi.org/10.1107/S1399004715000826" target="_blank">10.1107/S1399004715000826</a>.</p>
    
</div>

    
</div>

    
</div>


<div id="presentations" class="container">
    
        

<div class="row">
    
        

<div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">
    
        <h2>Presentations and Workshops</h2>

<h3>2018</h3>

<p><em>(workshop)</em> Croll, T. <em>ISOLDE. </em>Chimera and Cytoscape/STRING Course, Copenhagen, Denmark. 12 - 13 Nov, 2018</p>

<p><em>(presentation)</em> Croll, T. <em>ISOLDE: Macromolecular model building comes alive</em>. PHENIX developers' meeting, Berkeley, USA. Sep, 2018</p>

<p><em>(poster)</em> Croll, T., A. J. McCoy and R. Read. <em>ISOLDE: Bringing macromolecular model building to life</em>. 31st European Crystallography Meeting, Oviedo, Spain. 22 - 27 Aug, 2018</p>

<p><em>(workshop)</em> Croll, T. <em>ISOLDE</em>. Instruct Course on Model Building and Refinement for High Resolution EM Maps (4th Icknield Workshop), Diamond Light Source, Didcot, UK. 1 - 4 May, 2018</p>

<p><em>(presentation)</em> Croll, T. <em>ISOLDE: Macromolecular model building comes alive. </em>PHENIX developers' meeting, Santa Fe, USA. 26-29 Mar, 2016</p>

<h3>2017</h3>

<p><em>(presentation)</em> Croll, T. <em>ISOLDE: Interactive model building in a physically realistic environment. </em>Conference on methods and applications in the frontier between MX and CryoEM, Barcelona, Spain. 19 - 21 Sep, 2017</p>

<p><em>(presentation)</em> Croll, T. <em>ISOLDE: A next-generation environment for interactive model building in low-resolution maps.</em> CCP-EM/CCP-BIOSIM workshop, Leeds, UK. 14 Jul, 2017</p>

<p><em>(workshop)</em> Croll, T. <em>Low resolution refinement. </em>Erice School of Crystallography, Erice, Italy. 2 - 11 Jun, 2017</p>

<p><em>(presentation)</em> Croll, T. <em>ISOLDE: A next-generation environment for interactive model building in low-resolution maps. </em>CCP-EM Spring Symposium, Diamond Light Source, Didcot, UK. 24 - 26 Apr, 2017</p>

<h3>2016</h3>

<p><em>(presentation)</em> Croll, T. <em>Interactive molecular dynamics flexible fitting: a physically realistic environment for rapid and high-fidelity model building. </em>30th biennial meeting of the Society of Crystallographers of Australia and New Zealand, Hobart, Australia. 29 Mar - 1 Apr, 2016</p>

<p><em>(presentation)</em> Croll, T. <i>Interactive molecular dynamics flexible fitting: adding a sense of touch to structural biology. </i>Talks and demonstrations given at Aarhus University, Cambridge MRC Laboratory of Molecular Biology, Diamond Light Source and Oxford Structural Genomics Consortium, Jan 2016</p>
    
</div>

    
</div>

    
</div>


<div id="structures" class="container">
    
        

<div class="row">
    
        

<div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">
    
        <h2>Structures produced using ISOLDE</h2>
    
        

<div class="row">
    
        

<div class="col col-11 col-sm-4 col-md-3 col-lg-2 col-xl-2">
    
        








<img src="../media/filer_public_thumbnails/filer_public/d8/4e/d84e696c-6d1a-4b85-8d0d-05f61e1d63cd/6o1m.jpg__771x617_q85_subsampling-2.jpg"
    alt=""
    
    
    
    class="img-fluid rounded img-thumbnail"
>









    
</div>

    
        

<div class="col col-11 col-sm-7 col-md-8 col-lg-9 col-xl-9">
    
        <p>PDB IDs: <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6o1k">6o1k</a>, <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6o1l">6o1l</a>, <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6o1m">6o1m</a> (pictured: 6o1m)</p>

<p>Reference: Pei, X. Y., T. Dendooven, E. Sonnleitner, S. Chen, U. Bläsi, and B. F. Luisi . "Architectural principles for Hfq/Crc-mediated regulation of gene expression." <i>Elife</i>. 2019; 8:e43158. doi: <a href="https://doi.org/10.7554/eLife.43158">10.7554/eLife.43158</a></p>
    
</div>

    
</div>

    
        

<div class="row">
    
        

<div class="col col-11 col-sm-4 col-md-3 col-lg-2 col-xl-2">
    
        








<img src="../media/filer_public_thumbnails/filer_public/0e/7b/0e7b5aaa-6508-4b90-89d9-8071db9c7a08/6ogd.jpg__594x617_q85_subsampling-2.jpg"
    alt=""
    
    
    
    class="img-fluid rounded img-thumbnail"
>









    
</div>

    
        

<div class="col col-11 col-sm-7 col-md-8 col-lg-9 col-xl-9">
    
        <p>PDB ID: <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6ogd" target="_blank">6ogd</a></p>

<p>Reference: Piper, Sarah J., Lou Brillault, Rosalba Rothnagel, Tristan I Croll, Joseph K Box, Irene Chassagnon, Sebastian Scherer, Kenneth N Goldie, Sandra A Jones, Femke Schepers, Lauren Hartley-Tassell, Thomas Ve, Jason N Busby, Julie E Dalziel, J Shaun Lott, Ben Hankamer, Henning Stahlberg, Mark RH Hurst, and Michael J Landsberg. 2019. "Cryo-EM structures of the pore-forming A subunit from the Yersinia entomophaga ABC toxin." <em>Nature Communications </em>10 (1): 1952. doi: <a href="https://doi.org/10.1038/s41467-019-09890-8">10.1038/s41467-019-09890-8</a></p>
    
</div>

    
</div>

    
        

<div class="row">
    
        

<div class="col col-11 col-sm-4 col-md-3 col-lg-2 col-xl-2">
    
        








<img src="../media/filer_public_thumbnails/filer_public/76/14/7614fe87-7006-4a88-8bb1-5ae975587bf2/6i2m.jpg__646x686_q85_subsampling-2.jpg"
    alt=""
    
    
    
    class="img-fluid rounded img-thumbnail"
>









    
</div>

    
        

<div class="col col-11 col-sm-7 col-md-8 col-lg-9 col-xl-9">
    
        <p>PDB ID: <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6i2m">6i2m</a></p>

<p>Reference: Gao, C., M.A. Pallett, T.I. Croll, G.L. Smith and S.C. Graham, 2019. <em>Molecular basis of cullin-3 (Cul3) ubiquitin ligase subversion by vaccinia virus protein A55</em>. J. Biological Chemistry, in press. DOI: <a href="https://dx.doi.org/10.1074/jbc.RA118.006561" target="_blank"><span class="slug-metadata-note ahead-of-print"><span class="col-2-ahead-of-print-doi"><span class="slug-doi" title="10.1074/jbc.RA118.006561">10.1074/jbc.RA118.006561</span></span></span></a></p>
    
</div>

    
</div>

    
        

<div class="row">
    
        

<div class="col col-11 col-sm-4 col-md-3 col-lg-2 col-xl-2">
    
        








<img src="../media/filer_public_thumbnails/filer_public/bc/07/bc072b46-f293-4c15-9e2a-6e0ed40f7b7b/6g63.jpg__646x686_q85_subsampling-2.jpg"
    alt=""
    
    
    
    class="img-fluid rounded img-thumbnail"
>









    
</div>

    
        

<div class="col col-11 col-sm-7 col-md-8 col-lg-9 col-xl-9">
    
        <p>PDB ID: <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6g63" target="_blank">6g63</a></p>

<p>Reference: Bandyra, K.J., J.M Wandzik and B.F. Luisi, 2018. <em>Substrate recognition and autoinhibition in the central ribonuclease RNase E.</em> Mol. Cell. 72(2): 275-285.e4. DOI: <a class="ext active" href="http://dx.doi.org/10.1016/j.molcel.2018.08.039" target="_blank">10.1016/j.molcel.2018.08.039</a></p>
    
</div>

    
</div>

    
        

<div class="row">
    
        

<div class="col col-11 col-sm-4 col-md-3 col-lg-2 col-xl-2">
    
        








<img src="../media/filer_public_thumbnails/filer_public/82/c7/82c7b01d-2f0d-46fe-9cf4-6e9af3ce1352/6eyc.jpg__646x686_q85_subsampling-2.jpg"
    alt=""
    
    
    
    class="img-fluid rounded img-thumbnail"
>









    
</div>

    
        

<div class="col col-11 col-sm-7 col-md-8 col-lg-9 col-xl-9">
    
        <p>PDB ID: <a href="https://www.ebi.ac.uk/pdbe/entry/pdb/6eyc" target="_blank">6eyc</a></p>

<p>Reference: Croll, T.I. 2018, <em>ISOLDE: a physically realistic environment for model building into low-resolution electron density maps. </em>Acta Crystallographica D74: 519-530. DOI: <a class="ext active" href="http://dx.doi.org/10.1107/S2059798318002425" target="_blank">10.1107/S2059798318002425</a></p>
    
</div>

    
</div>

    
</div>

    
</div>

    
</div>

                            
						</div>
					</div>
				</div>
			</section>



        

<div class="row">
    
        

<div class="col col-1 col-sm-1 col-md-1 col-lg-1 col-xl-1">
    
</div>

    
        

<div class="col col-10 col-sm-10 col-md-10 col-lg-10 col-xl-10">
    
        <p>ISOLDE is developed and maintained by <a href="mailto:tcroll@altoslabs.com" class>Tristan Croll</a> at <a href="https://altoslabs.com">Altos Labs</a>. Development began in the <a href="https://www.cimr.cam.ac.uk/staff/professor-randy-j-read-frs" target="_blank" class>lab of Randy Read</a> at the <a href="https://www.cimr.cam.ac.uk/" target="_blank">Cambridge Institute for Medical Research</a>, supported by Prof. Read's Wellcome Trust Principal Research Fellowship 209407/Z/17/Z. Early development and testing was been aided by the generous donation of a GTX Titan Xp GPU by NVIDIA Corporation.</p>
    
</div>

    
</div>

        

<div class="row">
    
        

<div class="col col-1 col-sm-1 col-md-1 col-lg-1 col-xl-1">
    
</div>

    
        

<div class="col col-10 col-sm-10 col-md-10 col-lg-10 col-xl-10">
    
        <p>Website design by Tristan Croll based on <a href="https://templated.co/hielo" target="_blank">Hielo by Templated</a> under a CC BY 3.0 license. Powered by <a href="https://www.django-cms.org/en/" target="_blank" class>Django-CMS</a>.<br>
<a href="https://www.cam.ac.uk/about-this-site/privacy-policy">Privacy policy</a></p>
    
</div>

    
</div>

        <footer id="footer">
            <div class="container">
                <ul class="icons">
                    <li><a href="https://twitter.com/intent/tweet?text=ISOLDE%3A+bringing+structural+biology+to+life.+https://tristanic.github.io/isolde"
                        class="icon fa-twitter" target="_blank"><span class="label">Twitter</span></a></li>
                    <li><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A//www.isolde.cam.ac.uk"
                        class="icon fa-facebook" target="_blank"><span class="label">Facebook</span></a></li>
                    <li><a href="https://www.linkedin.com/shareArticle?mini=true&url=https://tristanic.github.io/isolde&title=ISOLDE%3A%20Bringing%20structural%20biology%20to%20life.&source=ISOLDE"
                        class="icon fa-linkedin" target="_blank"><span class="label">LinkedIn</span></a></li>
                    <li><a href="mailto:?subject=ISOLDE for molecular model building&amp;body=Check out this cool model building package%3A https://tristanic.github.io/isolde."
                        class="icon fa-envelope-o"><span class="label">Email</span></a></li>
                </ul>
            </div>
        </footer>

        <!-- Hielo template code -->
        <script src="../static/isolde/assets/js/jquery.min.js"></script>
        <script src="../static/isolde/assets/js/jquery.scrollex.min.js"></script>
        <script src="../static/isolde/assets/js/skel.min.js"></script>
        <script src="../static/isolde/assets/js/util.js"></script>
        <script src="../static/isolde/assets/js/main.js"></script>
        <script src="../static/isolde/node_modules/jquery/dist/jquery.min.js"></script>
        <script src="../static/isolde/node_modules/bootstrap/dist/js/bootstrap.min.js"></script>


        

        
    </body>
</html>
