
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Manipulating your model &#8212; ISOLDE 1.4 documentation</title>
    <link rel="stylesheet" type="text/css" href="../../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/alabaster.css" />
    <link rel="stylesheet" type="text/css" href="../../_static/css/custom.css" />
    <script data-url_root="../../" id="documentation_options" src="../../_static/documentation_options.js"></script>
    <script src="../../_static/jquery.js"></script>
    <script src="../../_static/underscore.js"></script>
    <script src="../../_static/doctools.js"></script>
    <script src="../../_static/js/external_links.js"></script>
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Monitoring the quality of your model" href="validate.html" />
    <link rel="prev" title="Basic simulation controls" href="sim_settings.html" />
   
  <link rel="stylesheet" href="../../_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="manipulating-your-model">
<span id="id1"></span><h1>Manipulating your model<a class="headerlink" href="rebuild.html#manipulating-your-model" title="Permalink to this headline">¶</a></h1>
<div class="toctree-wrapper compound">
</div>
<div class="contents local topic" id="contents">
<ul class="simple">
<li><p><a class="reference internal" href="rebuild.html#overview" id="id6">Overview</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#peptide-flip-and-rotamer-adjustment-buttons" id="id7">Peptide flip and rotamer adjustment buttons</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#the-restraints-tab" id="id8">The Restraints tab</a></p>
<ul>
<li><p><a class="reference internal" href="rebuild.html#position-restraints" id="id9">Position Restraints</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#simple-distance-restraints" id="id10">Simple Distance Restraints</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#secondary-structure" id="id11">Secondary Structure</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#register-shift-protein" id="id12">Register Shift Protein</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#reference-models" id="id13">Reference Models</a></p>
<ul>
<li><p><a class="reference internal" href="rebuild.html#options" id="id14">Options</a></p>
<ul>
<li><p><a class="reference internal" href="rebuild.html#distance-restraints" id="id15">Distance restraints</a></p></li>
<li><p><a class="reference internal" href="rebuild.html#torsion-restraints" id="id16">Torsion restraints</a></p></li>
</ul>
</li>
</ul>
</li>
<li><p><a class="reference internal" href="rebuild.html#manage-release-adaptive-restraints" id="id17">Manage/Release Adaptive Restraints</a></p></li>
</ul>
</li>
</ul>
</div>
<section id="overview">
<h2><a class="toc-backref" href="rebuild.html#id6">Overview</a><a class="headerlink" href="rebuild.html#overview" title="Permalink to this headline">¶</a></h2>
<p>Where simple tugging on atoms is insufficient to solve problems with your model,
ISOLDE provides various tools to help. Some of the most commonly used live up on
the ribbon menu:</p>
<figure class="align-default">
<img alt="../../_images/rotarama_buttons.png" src="../../_images/rotarama_buttons.png" />
</figure>
<p>... while the remainder are found in ISOLDE’s Restraints tab. Let’s start with the
ribbon menu buttons.</p>
</section>
<section id="peptide-flip-and-rotamer-adjustment-buttons">
<h2><a class="toc-backref" href="rebuild.html#id7">Peptide flip and rotamer adjustment buttons</a><a class="headerlink" href="rebuild.html#peptide-flip-and-rotamer-adjustment-buttons" title="Permalink to this headline">¶</a></h2>
<p>The two “Flip” buttons will become active when you select a single amino acid
residue that isn’t at the N-terminus of a fragment (that is, has another amino
acid bonded to its backbone nitrogen). The “Flip peptide” button uses temporary
torsion restraints on <em>phi</em> and <em>psi</em> to (attempt to) flip the N-terminal
peptide bond by 180 degrees. The “Flip cis&lt;-&gt;trans” button, meanwhile, flips the
target of the always-present restraint on <em>omega</em> to switch from <em>cis</em> to
<em>trans</em> or vice versa. If no simulation is running, a small local one will be
automatically started to do the job.</p>
<p>The “Preview next” rotamer button, meanwhile, becomes active whenever you select
a single rotameric amino acid residue. Clicking it once will create a preview of
the most common rotamer for that residue, overlaid on the model with a label
giving its name and prevalence. Clicking again will give the next most common
residue and so on:</p>
<figure class="align-default">
<img alt="../../_images/rotamer_preview.jpg" src="../../_images/rotamer_preview.jpg" />
</figure>
<p>When you have found what you believe to be the most suitable rotamer, you have two
choices:</p>
<ol class="loweralpha simple">
<li><p>click the “Set coords” button to tell ISOLDE to simply set the model coordinates
to match the preview. If a simulation is running, this will automatically trigger
a quick energy minimisation before continuing. <strong>(Do NOT use this option if the
preview coordinates clash severely with surrounding atoms. Instead, use option b)</strong></p></li>
<li><p>click the “Restrain” button to add torsion restraints forcing the sidechain <em>chi</em>
torsions to match the preview. This approach is “safe” in that it will never
introduce severe clashes - but bulky sidechains will usually need a little help
via mouse tugging to make large changes.</p></li>
</ol>
<p>All user-applied dihedral restraints (other than the always-on restraints on the
peptide bond <em>omega</em> dihedrals) are displayed on the model using a ring-and-post
motif as shown for this lysine rotamer:</p>
<figure class="align-default" id="id4">
<img alt="Dihedral restraint visualisation" src="../../_images/dihedral_restraints.png" />
<figcaption>
<p><span class="caption-text">The difference between the current dihedral angle and the target is
indicated by the angle between the posts, and the level of satisfaction of
the restraints is further indicated by colour. The thickness of each
indicator is proportional to the strength of the restraint.</span><a class="headerlink" href="rebuild.html#id4" title="Permalink to this image">¶</a></p>
</figcaption>
</figure>
<p>The final button (“Release”) can be used to disable any torsion restraints added by
choosing option (b).</p>
</section>
<section id="the-restraints-tab">
<span id="isolde-restraints-tab"></span><h2><a class="toc-backref" href="rebuild.html#id8">The Restraints tab</a><a class="headerlink" href="rebuild.html#the-restraints-tab" title="Permalink to this headline">¶</a></h2>
<figure class="align-default">
<img alt="../../_images/restraints_tab_overview.png" src="../../_images/restraints_tab_overview.png" />
</figure>
<section id="position-restraints">
<h3><a class="toc-backref" href="rebuild.html#id9">Position Restraints</a><a class="headerlink" href="rebuild.html#position-restraints" title="Permalink to this headline">¶</a></h3>
<figure class="align-default">
<img alt="../../_images/position_restraint_toolbar.png" src="../../_images/position_restraint_toolbar.png" />
</figure>
<p>The three buttons in this widget allow you to, in order:</p>
<ul class="simple">
<li><p>pin each heavy atom in the current selection to its current position;</p></li>
<li><p>pin a single heavy atom to the current position of the pivot indicator; and</p></li>
<li><p>release any existing position restraints on the current selection.</p></li>
</ul>
<p>The applied restraint behaves for most purposes like a simple harmonic spring
(that is, the applied force is proportional to distance), but transitions to
a constant force at a maximum-force threshold to avoid instability. The spring
constant for added restraints is set in the field on the right (changes affect
only future restraints, not existing ones).</p>
</section>
<section id="simple-distance-restraints">
<h3><a class="toc-backref" href="rebuild.html#id10">Simple Distance Restraints</a><a class="headerlink" href="rebuild.html#simple-distance-restraints" title="Permalink to this headline">¶</a></h3>
<figure class="align-default">
<img alt="../../_images/distance_restraint_toolbar.png" src="../../_images/distance_restraint_toolbar.png" />
</figure>
<p>The first button is straightforward: it adds a harmonic distance restraint between
two selected heavy atoms, with the target distance and spring constant defined
by the relevant input fields. The “Release (internal)” button will release all
distance restraints for which <strong>both</strong> endpoint atoms are currently selected.
The “Release (any)” button releases distance restraints for which <strong>either</strong>
endpoint atom is selected. Over on the right, the “Use current distance” button
changes the “Target” field to the current distance between atoms when you have
exactly two heavy atoms selected.</p>
<p>Simple distance restraints appear in the model as double-ended pistons:</p>
<figure class="align-default" id="id5">
<img alt="Distance restraint visualisations" src="../../_images/distance_restraints.png" />
<figcaption>
<p><span class="caption-text">The length of the central cylinder indicates the target distance, while its
thickness indicates the strength of the restraint.</span><a class="headerlink" href="rebuild.html#id5" title="Permalink to this image">¶</a></p>
</figcaption>
</figure>
</section>
<section id="secondary-structure">
<h3><a class="toc-backref" href="rebuild.html#id11">Secondary Structure</a><a class="headerlink" href="rebuild.html#secondary-structure" title="Permalink to this headline">¶</a></h3>
<figure class="align-default">
<img alt="../../_images/secondary_structure_toolbar.png" src="../../_images/secondary_structure_toolbar.png" />
</figure>
<p>The first three buttons in this widget apply a combination of torsion and distance
restraints to the current protein selection to guide the model towards your
desired secondary structure. In brief:</p>
<ul class="simple">
<li><p>the selection is broken into contiguous fragments;</p></li>
<li><p>within each fragment, torsion restraints are added to the <em>phi</em> and <em>psi</em>
dihedrals, and distance restraints between each O-(N+4) and CA-(CA+2) pair,
with target values matching the desired secondary structure;</p></li>
<li><p>for fragments assigned as beta strands, any backbone H-bonds to surrounding
strands are reinforced by distance restraints.</p></li>
</ul>
<p>The first two buttons simply set everything in the selection to beta strand
or alpha helix respectively. The third assigns restraints based on the current
secondary structure annotation. <em>(NOTE: if this wasn’t defined in your input
file, ChimeraX automatically assigns secondary structure using the DSSP algorithm.
This is sometimes a bit over-enthusiastic in what it considers to be a helix or
strand - so if you use this button you may find yourself subsequently needing to
release quite a few residues.)</em></p>
<p>The final button will, of course, release all secondary structure restraints on
the selected atoms.</p>
</section>
<section id="register-shift-protein">
<h3><a class="toc-backref" href="rebuild.html#id12">Register Shift Protein</a><a class="headerlink" href="rebuild.html#register-shift-protein" title="Permalink to this headline">¶</a></h3>
<figure class="align-default">
<img alt="../../_images/register_shift_toolbar.png" src="../../_images/register_shift_toolbar.png" />
</figure>
<p>In models built into low-resolution maps (particularly older “legacy” models
from the wwPDB) it is a reasonably common occurrence for part of the model
sequence to get out of step with the map - that is, while the overall fold
topology is correct, the wrong residue is at each site. The register shift
tool is designed to help with that. Given a selection defining a single
contiguous stretch of amino acid residues, it uses a set of moving position
restraints on the N, CA, C and (where present) CB atoms to tug the backbone
smoothly along splines fitted through these atoms’ initial positions,
shifting them by the number of residue positions you request. Shifting in
this way avoids having to deal with the tangle of sidechain-sidechain clashes
that can be the result of other methods. Once the shift is done, clicking the
Release button will remove the position restraints.</p>
</section>
<section id="reference-models">
<h3><a class="toc-backref" href="rebuild.html#id13">Reference Models</a><a class="headerlink" href="rebuild.html#reference-models" title="Permalink to this headline">¶</a></h3>
<figure class="align-default">
<img alt="../../_images/reference_model_widget1.png" src="../../_images/reference_model_widget1.png" />
</figure>
<p>This widget allows you to use an existing model (experimental or predicted) as
reference for the assignment of ISOLDE’s
<a class="reference external" href="help:user/commands/isolde_restrain.html">adaptive distance and torsion restraints</a>.</p>
<p>After choosing a model in the “Reference model:” drop-down menu, the table
below will automatically populate with possible matches between each chain
in the working model and each chain in the reference model. The reference model
(with colours shifted visually to distinguish it from the working model)
will be aligned to the working model using <a class="reference external" href="help:user/commands/matchmaker.html">MatchMaker</a>. If the reference
model was generated by AlphaFold2 (or any other prediction method that
produces a Predicted Aligned Error (PAE) matrix in the same format), you
can load that using the button at top right to enable confidence-based
assignment of distance restraint parameters. <em>(NOTE: to use a PAE matrix,
the reference model must be the complete, unchanged prediction, with no
residues added or removed.)</em></p>
<p>In the table, the “Chain” column will have an entry for each chain in your
working model with at least one match (identity greater than 50%) in the
reference model. Each potential match for that chain will be listed in the
following rows, ordered by RMSD in the current alignment. The “Ref” column
gives the chain ID of each refererence chain. The “Align” button allows you
to re-align the reference model based on that (working chain):(reference chain)
combination. The checkboxes in the “Distances” and “Torsion” columns allow
you to specify this reference chain for the corresponding restraints (of course,
you can only specify one reference chain for each working chain - while if you
like you <em>can</em> choose different chains for torsion and distance restraints,
but this is generally not a good idea). The final three columns provide
important information about the relationship between working and reference
chains: the fraction of the working sequence covered by the reference sequence;
the sequence identity within that fraction; and the C-alpha RMSD in the current
alignment. The Apply buttons below will create restraints based on the currently
ticked checkboxes. The “Apply (selected residues only)” button, as the name
suggests, will assign restraints only to those residues with atoms currently
selected in the main ChimeraX window. <em>(NOTE: once the restraints are assigned,
they no longer rely on the reference model in any way. You can safely close it
if you wish)</em></p>
<p>By default, when using a multi-chain model as reference, inter-chain distance
restraints will be generated to maintain protein-protein interfaces. This can
be disabled if desired via the Options widget (see below). Be sensible: this
will only yield a reasonable result if each chosen reference chain has a
low CA-RMSD in the current alignment. If the same chain is used as reference
for two different working chains this option will be ignored, and inter-chain
restraints will <em>not</em> be generated.</p>
<section id="options">
<h4><a class="toc-backref" href="rebuild.html#id14">Options</a><a class="headerlink" href="rebuild.html#options" title="Permalink to this headline">¶</a></h4>
<figure class="align-default">
<img alt="../../_images/reference_model_options1.png" src="../../_images/reference_model_options1.png" />
</figure>
<p>This sub-widget allows you to adjust the assignment behaviour and properties
of reference restraints before you apply them. <em>(NOTE: changes here do not
affect any existing restraints.)</em> The left-hand column is dedicated to
distance restraints and the right to torsion restraints.</p>
<section id="distance-restraints">
<h5><a class="toc-backref" href="rebuild.html#id15">Distance restraints</a><a class="headerlink" href="rebuild.html#distance-restraints" title="Permalink to this headline">¶</a></h5>
<p><strong>Inter-chain restraints</strong> checkbox: if checked, restraints will be created
across chain-chain interfaces if possible. Otherwise, only intra-chain
restraints will be created.</p>
<p><strong>Adjust by PAE</strong> checkbox: if checked and a valid PAE matrix has been assigned
to the reference model, then the properties of each distance restraint will be
adjusted according to the predicted aligned error between the residues containing
the restrained atoms. See the <a class="reference internal" href="../../commands/isolde_restrain.html#isolde-restrain-distances-cmd"><span class="std std-ref">isolde restrain distances</span></a> command for more
detals.</p>
<p><strong>Use rigid-body matching</strong> checkbox: if checked, restraints will be applied only
to those parts of the working model which make a good rigid-body match to the
reference model. This is recommended if you are not using a PAE matrix. If
unchecked, only the sequence alignment will be used for restraint assignment.</p>
<p><strong>Overall strength and Fuzziness</strong> sliders: these allow you to adjust the
overall strength of the restraints, and how quickly their applied force falls
off with distance outside the harmonic region. The plot below gives you a
preview of the resulting force profile for a restraint with a target distance of
3 Angstroms and a “perfect” PAE value of 0.2.</p>
</section>
<section id="torsion-restraints">
<h5><a class="toc-backref" href="rebuild.html#id16">Torsion restraints</a><a class="headerlink" href="rebuild.html#torsion-restraints" title="Permalink to this headline">¶</a></h5>
<p><strong>Restrain sidechain</strong> checkbox: uncheck this if you wish to restrain only the
protein backbone, while leaving sidechains free to move.</p>
<p><strong>Adjust by pLDDT</strong> checkbox: if checked, values in the B-factor column of the
reference model will be subjected to a basic sanity check to make sure they look
like pLDDT values (after some basic checks to see if (a) the values fall in the
range 0..1 or 0..100, and (b) values are identical for all atoms in each
residue, and (c) the median value for unstructured residues is lower than the
median for helix/strand residues). If this check fails a warning will pop up
suggesting you disable this option before continuing. Otherwise, the parameters
for each torsion restraint will be adjusted based on the pLDDT for that
residue. See the <a class="reference internal" href="../../commands/isolde_restrain.html#adaptive-dihedral-restraint-cmd"><span class="std std-ref">isolde restrain torsions</span></a> command for more details.</p>
<p><strong>Overall strength and Fuzziness</strong> sliders: these allow you to adjust the
overall strength of the restraints, and how quickly their applied force falls
off with angle deviation outside the harmonic region. The plot below gives you a
preview of the resulting force profile for a restraint with pLDDT=100.</p>
</section>
</section>
</section>
<section id="manage-release-adaptive-restraints">
<h3><a class="toc-backref" href="rebuild.html#id17">Manage/Release Adaptive Restraints</a><a class="headerlink" href="rebuild.html#manage-release-adaptive-restraints" title="Permalink to this headline">¶</a></h3>
<figure class="align-default">
<img alt="../../_images/manage_restraints_widget2.png" src="../../_images/manage_restraints_widget2.png" />
</figure>
<p>This widget allows you to selectively release adaptive distance and/or torsion
restraints added via the Reference Models widget or via the <code class="docutils literal notranslate"><span class="pre">isolde</span> <span class="pre">restrain</span></code>
commands. Additionally, here is where you can change their colour schemes to
your liking, and adjust the display threshold on distance restraints to show
only those restraints which are currently strained.</p>
<p>At present, the “Distance restraints group” drop-down menu is primarily a
future-proofing addition ahead of planned support for distance restraints from
other sources such as cross-linking/mass spectrometry experiments. If you really
wish to, you can use the <code class="docutils literal notranslate"><span class="pre">isolde</span> <span class="pre">restrain</span> <span class="pre">distances</span></code> command to create
multiple independent groups of reference-model difference restraints, but the
value of such an endeavour is unclear.</p>
<p>The “Colours” button displays the current colour scheme as three horizontal
stripes, with the “overly compressed” colour at bottom, the “optimally
satisfied” colour in the middle, and “overly stretched” at top. Clicking it will
bring up a series of three colour chooser dialogues, allowing you to choose new
colours in this order. The “Reset” button beside it resets the colour scheme to
the stored defaults.</p>
<p>Dragging the “Display threshold” slider to the right will progressively hide
the most satisfied distance restraints to cut down on visual clutter. When the
slider is all the way to the left, all distance restraints will be shown.</p>
<p>The four buttons below this offer different options for releasing restraints
with varying levels of selectivity. The “Release all selected” button will
release all restraints for which <strong>at least one</strong> endpoint atom is selected. “Release
surrounding” will release restraints with <strong>exactly one</strong> endpoint atom selected.
“Release internal” will release only restraints with <strong>both</strong> endpoint atoms
selected. Finally, the “Remove all” button will remove <strong>all</strong> restraints in
this group.</p>
<p>The remaining buttons apply to torsion restraints. The “Colours” button here
behaves analogously to the one for distance restraints; clicking it allows you
to pick the colours for severely strained, moderately strained, and satisfied
restraints in that order.</p>
<p>Unlike the distance restraint release buttons, the torsion restraint equivalents
act on <strong>whole residues</strong> - every residue with at least one atom selected will be
affected. As the names suggest, the buttons allow you to release either all
restraints (<em>phi</em>, <em>psi</em> and <em>chi</em><sub>n</sub>) or only sidechain restraints (<em>chi</em><sub>n</sub>)
on the selected residues, or to delete all adaptive torsion restraints in the
model.</p>
</section>
</section>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
<h3><a href="../../isolde.html">Table of Contents</a></h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../isolde.html">Home</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="../ISOLDE.html">The ISOLDE GUI</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="../intro.html">What is ISOLDE all about?</a></li>
<li class="toctree-l2"><a class="reference internal" href="../requirements.html">System Requirements</a></li>
<li class="toctree-l2 current"><a class="reference internal" href="index.html">GUI Details</a></li>
<li class="toctree-l2"><a class="reference internal" href="../../tutorials/isolde.html">Tutorials</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../commands/isolde_restrain.html">Adaptive Restraint Schemes</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../commands/isolde_remote.html">Remote Control Options</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../commands/isolde.html">The ISOLDE Command Line</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../api/isolde.html">ISOLDE Python API</a></li>
<li class="toctree-l1"><a class="reference internal" href="../license.html">License</a></li>
<li class="toctree-l1"><a class="reference internal" href="../citing.html">Citing ISOLDE</a></li>
<li class="toctree-l1"><a class="reference internal" href="../contact.html">Contact Info</a></li>
</ul>
<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="../../isolde.html">Documentation overview</a><ul>
  <li><a href="../ISOLDE.html">The ISOLDE GUI</a><ul>
  <li><a href="index.html">GUI Details</a><ul>
      <li>Previous: <a href="sim_settings.html" title="previous chapter">Basic simulation controls</a></li>
      <li>Next: <a href="validate.html" title="next chapter">Monitoring the quality of your model</a></li>
  </ul></li>
  </ul></li>
  </ul></li>
</ul>
</div>
<div id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="../../search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</div>
<script>$('#searchbox').show(0);</script>
        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2016-2022, Tristan Croll.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 4.3.2</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.12</a>
      
      |
      <a href="../../_sources/tools/gui/rebuild.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>