<!DOCTYPE html>

<html lang="en" prefix="og: http://ogp.me/ns#">
<head>
    <title>Case study: 3loh</title>
    <meta charset="utf-8"/>
    <meta name="viewport" content="width=device-width, inital-scale=1"/>


    <meta name="twitter:card" content="summary"/>
    <meta name="twitter:title" content="ISOLDE: Bringing structural biology to life"/>
    <meta name="twitter:description"
          content="Fast and flexible interactive molecular modelling into experimental maps"/>
    <meta name="twitter:image" content="/isolde/static/isolde/images/logo_square.jpg"/>

    <meta property="og:title" content="ISOLDE: Bringing structural biology to life"/>
    <meta property="og:image" content="/isolde/static/isolde/images/logo_square.jpg"/>
    <meta property="og:description" content="Fast and flexible interactive molecular modelling into experimental maps"/>
    <meta property="og:url" content="https://tristanic.github.io/isolde/case-studies/case-study-3loh/"/>


    <link href="../../static/isolde/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
    <link href="../../static/isolde/assets/css/main.css" rel="stylesheet" type="text/css"/>


</head>


<body class="subpage">


<header id="header">
    <div class="logo"><a href="../../index.html">ISOLDE <span>home</span></a></div>
    <a href="#menu">Menu</a>
</header>

<!-- Nav -->
<nav id="menu">
    <ul class="links">


        <li class="child ancestor">
            <a href="../../index.html">Home</a>

            <ul>


                <li class="child">
                    <a href="../../what-isolde/index.html">What is ISOLDE?</a>

                </li>

                <li class="child">
                    <a href="../../download/index.html">Download</a>

                </li>

                <li class="child">
                    <a href="../../features-main/index.html">Features</a>

                    <ul>


                        <li class="child">
                            <a href="../../features-main/features-graphics/index.html">Graphics</a>

                        </li>

                        <li class="child">
                            <a href="../../features-main/features-interactive-md/index.html">Interactive MD</a>

                        </li>

                        <li class="child">
                            <a href="../../features-main/real-time-validation/index.html">Real-time Validation</a>

                        </li>

                        <li class="child">
                            <a href="../../features-main/live-atomic-symmetry/index.html">Live atomic symmetry</a>

                        </li>

                        <li class="child">
                            <a href="../../features-main/interactive-restraints/index.html">Interactive restraints</a>

                        </li>

                        <li class="child">
                            <a href="../../features-main/real-time-crystallographic-map-calculations/index.html">Real-time
                                crystallographic map calculations</a>

                        </li>

                    </ul>

                </li>

                <li class="child">
                    <a href="../../faq/index.html">Frequently Asked Questions</a>

                </li>

                <li class="child">
                    <a href="../../documentation/index.html">Documentation</a>

                </li>

                <li class="child">
                    <a href="../../performance-benchmarks/index.html">Performance</a>

                </li>

                <li class="child">
                    <a href="../../about/index.html">About</a>

                </li>

                <li class="child">
                    <a href="../../licensing/index.html">Licensing</a>

                </li>

                <li class="child">
                    <a href="../../publications/index.html">Publications</a>

                </li>

                <li class="child ancestor">
                    <a href="../index.html">Case Studies</a>

                    <ul>


                        <li class="child sibling">
                            <a href="../case-study-3io0/index.html">3io0</a>

                        </li>

                        <li class="child selected">
                            <a href="index.html">3loh</a>

                        </li>

                        <li class="child sibling">
                            <a href="../case-study-3j9d/index.html">3j9d</a>

                        </li>

                    </ul>

                </li>

                <li class="child">
                    <a href="../../contact/index.html">Contact</a>

                </li>

            </ul>

        </li>

    </ul>
</nav>


<!-- One -->
<section id="One" class="wrapper style3">
    <div class="inner">
        <header class="align-center">
            <p>ISOLDE: Bringing structural biology to life</p>
            <h2>Case study: 3loh</h2>
        </header>
    </div>
</section>

<!-- Two -->
<section id="two" class="wrapper style2">
    <div class="inner">
        <div class="box">
            <div class="content">

                <header class="align-center">

                </header>


                <div class="container">


                    <div class="row">


                        <div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">

                            <h2>3loh: Human insulin receptor ectodomain in complex with two F(ab) fragments per
                                monomer</h2>

                            <h3>3.8Å crystal, 1666 protein residues (data and original model: <a
                                    href="https://www.nature.com/articles/nature05106">McKern <em>et al.</em>, 2006</a>;
                                3loh model based on a later revision by  <a
                                        href="https://www.pnas.org/content/107/15/6771.short">Smith <em>et al.</em>,
                                    2010</a>)</h3>

                            <p><em>(The videos below recapitulate corrections originally made in collaboration with Mike
                                Lawrence and published in <a
                                        href="https://www.sciencedirect.com/science/article/pii/S0969212616000071">Croll </a></em><a
                                    href="https://www.sciencedirect.com/science/article/pii/S0969212616000071">et
                                al.</a><em><a
                                    href="https://www.sciencedirect.com/science/article/pii/S0969212616000071">,
                                2016</a>. I am very grateful for Mike's help and support throughout this journey)</em>.
                            </p>

                            <p>At the time this crystal was first solved, high-resolution structures of the first three
                                insulin receptor domains and many homologous antibody F(ab) fragments already existed -
                                but no template structure existed for the three fibronectin type-III (FnIII) domains
                                constituting the "leg" of the receptor. These had to be painstakingly hand-built into
                                the density with the aid of published - but contradictory - secondary structure
                                predictions. In the early 2000s (and still today) hand building into density this fuzzy
                                was a task not for the faint of heart! At 3.8Å, most sidechains appear as little more
                                than nondescript bumps on the backbone (or disappear entirely), and loops are typically
                                reduced to featureless tubes. On top of this, the achingly low data:parameter ratio
                                meant that overfitting and model bias (the unavoidable tendency for the maps to always
                                look a <em>bit</em> like the current model) was a constant challenge. As a result,
                                substantial errors in structures first solved at this resolution are a common
                                phenomenon:</p>

                            <p>


                                <img src="../../media/filer_public_thumbnails/filer_public/d6/57/d657bf95-0458-4716-bf0a-6719c6b965f9/isolde_pdb_quality_figure.jpg__1800x900_q85_subsampling-2.jpg"
                                     alt=""


                                     srcset="../../media/filer_public_thumbnails/filer_public/d6/57/d657bf95-0458-4716-bf0a-6719c6b965f9/isolde_pdb_quality_figure.jpg__576x576_q85_subsampling-2.jpg_576x576_q85_subsampling-2.jpg 576../../media/filer_public_thumbnails/filer_public/d6/57/d657bf95-0458-4716-bf0a-6719c6b965f9/isolde_pdb_quality_figure.jpg__768x768_q85_subsampling-2.jpgg__768x768_q85_subsampling-2.jpg 7../../media/filer_public_thumbnails/filer_public/d6/57/d657bf95-0458-4716-bf0a-6719c6b965f9/isolde_pdb_quality_figure.jpg__992x992_q85_subsampling-2.jpgjpg__992x992_q85_subsampling-2../../media/filer_public_thumbnails/filer_public/d6/57/d657bf95-0458-4716-bf0a-6719c6b965f9/isolde_pdb_quality_figure.jpg__1800x900_q85_subsampling-2.jpg.jpg__1800x900_q85_subsampling-2.jpg 1800w
        "
                                     sizes="
            
                (max-width: 576px) 576px,
            
                (max-width: 768px) 768px,
            
                (max-width: 992px) 992px,
            
            1800px
        "

                                     class="img-fluid rounded img-thumbnail"
                                >


                            </p>

                            <p><em>Above: Figure 6a from ISOLDE's inaugural paper <a
                                    href="https://onlinelibrary.wiley.com/iucr/doi/10.1107/S2059798318002425">(Croll,
                                2018)</a> plotting rate of stereochemical error as MolProbity score (lower is better) on
                                the y axis against fit to data as </em>R<sub>free</sub> <em>(again, lower is better) for
                                all 3.5-4.0Å crystal structures published between 2006 and October 2016. The first
                                observation to note is that there is little to no correlation between </em>R<em> factors
                                and model quality at these resolutions. The second is that the average error rate is
                                high - keep in mind that the MolProbity score is related to </em>ln<em>(error rate), and
                                a score close to 1 is typical for atomic-resolution models. </em></p>

                            <p>Despite the fact that substantial errors arise in many structures of this resolution,
                                this one will always hold a special place in my heart. As discussed in the <a
                                        href="../../about/index.html" class>About section</a>, it was my discovery of
                                the error shown below in the FnIII-3 domain that led me into the endlessly fascinating
                                world of structural biology.</p>

                            <h3>Notes on general model preparation and remodelling</h3>

                            <p>It's a simple trueism that an atomic B-factor (a stand-in for the mobility of an atom,
                                dictating how "smeared out" its density is in the crystal) can only be meaningful if the
                                atom is in the correct place to start with. When rebuilding a low-resolution model,
                                therefore, it is often advantageous to be very conservative with B-factors to reduce
                                overfitting effects. In this case I have removed the original TLS parameterisation for
                                the time being, and reset all atomic B-factors to the Wilson B (about 153Å<sup>2</sup>
                                for this crystal). While this initially drives the R-factors on equilibration much
                                higher than when the original B-factor model is used
                                (<em>R<sub>work</sub>/R<sub>free</sub></em> about 0.36/0.41 vs 0.32/0.36), the potential
                                for model bias (and development of artefacts following large rearrangements) is
                                significantly reduced. Each of the videos below derives from this starting model, and
                                each has the corrections from any previous videos applied. After applying all the
                                corrections shown, restrained refinement in <em>phenix</em>.<em>refine</em> (isotropic
                                B-factors only, no TLS) reduced <em>R<sub>work</sub>/R<sub>free</sub></em><em> </em>to
                                0.258/0.309, with a MolProbity score of 1.86 (down from 4.12). This could undoubtedly be
                                improved further with another round of inspection and rebuilding.</p>

                        </div>


                    </div>


                    <div class="row">


                        <div class="col col-11 col-sm-11 col-md-5 col-lg-5 col-xl-5">


                            <!-- START HTML5 VIDEO -->
                            <div class="Rvideo-box sixteen-nine">


                                <iframe src="https://www.youtube.com/embed/SS1d0xSXQZs" frameborder="0"
                                        allowfullscreen="true" width="100" height="75"></iframe>


                            </div>
                            <!-- END HTML5 VIDEO -->


                        </div>


                        <div class="col col-11 col-sm-11 col-md-6 col-lg-6 col-xl-6">

                            <h2>F(ab) chain D - simple 1-residue register shift</h2>

                            <p>This is a good illustration of one of ISOLDE's primary goals: to make simple problems
                                simple to fix. Here we have a short loop (forming one of the complementarity-determining
                                regions) that has been built one residue out of register (meaning that what points
                                inwards should point outwards, and vice versa). Once seen, it's fairly obvious what
                                needs to be done about it, and ISOLDE's register shifter makes short work of the
                                issue.</p>

                        </div>


                    </div>


                    <div class="row">


                        <div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">

                            <hr>

                        </div>


                    </div>


                    <div class="row">


                        <div class="col col-11 col-sm-11 col-md-5 col-lg-5 col-xl-5">


                            <!-- START HTML5 VIDEO -->
                            <div class="Rvideo-box sixteen-nine">


                                <iframe src="https://www.youtube.com/embed/AXRkTFep2XU" frameborder="0"
                                        allowfullscreen="true" width="100" height="75"></iframe>


                            </div>
                            <!-- END HTML5 VIDEO -->


                        </div>


                        <div class="col col-11 col-sm-11 col-md-6 col-lg-6 col-xl-6">

                            <h3>FnIII-1: large register shifts in two beta strands</h3>

                            <p>Residues 500~540 of the insulin receptor constitute two beta strands and a large
                                intervening loop. The beta strands constitute a substantial portion of a key receptor
                                dimer interface, while the loop as modelled places a number of conserved residues very
                                close to the insulin binding site. Furthermore, <a
                                        href="https://www.nature.com/articles/s41467-018-06826-6">new data published in
                                    2018</a> shows that much of this rregion is directly involved in insulin binding -
                                so correct modelling of it is thus rather important to our understanding of how the
                                receptor behaves. As originally published, the <em>N</em>- and <em>C</em>-terminal
                                strands were out of register by 4 and 6 residues respectively. Correcting them pulls the
                                intervening loop (containing one of the two intermolecular disulfides stabilising the
                                receptor dimer) tighter by ten residues, leaving it buried quite snugly in the dimer
                                interface. The extra residues in turn increase the size of the loops N- and C-terminal
                                to this stretch.</p>

                            <p>The large register shifts in this domain also strengthen the argument for a very
                                conservative B-factor parameterisation at this stage. Take residues Trp529-Val532 as an
                                example: in the original model these appear as part of a highly flexible loop, with
                                B-factors averaging 314Å<sup>2</sup>. After remodelling, they are shifted in register by
                                6 residues to instead form the <em>N</em>-terminus of a well-resolved beta-strand, with
                                B-factors after refining averaging 148Å<sup>2</sup>. Meanwhile, the residues that <em>were</em>
                                modelled as beta strand are pushed out to become part of a flexible loop. Attempting
                                this shift with the original B-factors creates a result that <em>looks</em> wrong, with
                                strong positive difference density along the beta strand and negative difference density
                                in the following loop. Until such time as ISOLDE gains a robust B-factor refinement
                                algorithm, my advice is to stick to the simplest practicable model until you are
                                confident your atomic coordinates are substantially correct.</p>

                        </div>


                    </div>


                    <div class="row">


                        <div class="col col-11 col-sm-11 col-md-11 col-lg-11 col-xl-11">

                            <hr>

                        </div>


                    </div>


                    <div class="row">


                        <div class="col col-11 col-sm-11 col-md-5 col-lg-5 col-xl-5">


                            <!-- START HTML5 VIDEO -->
                            <div class="Rvideo-box sixteen-nine">


                                <iframe src="https://www.youtube.com/embed/RtUo-FIbZEM" frameborder="0"
                                        allowfullscreen="true" width="100" height="75"></iframe>


                            </div>
                            <!-- END HTML5 VIDEO -->


                        </div>


                        <div class="col col-11 col-sm-11 col-md-6 col-lg-6 col-xl-6">

                            <h3>FnIII-3: complex register shift and rebuilding</h3>

                            <p>This is <em>the</em> site that started my journey. The combination of difficult density
                                (complicated by non-specific crystal contacts to two symmetry-related F(ab) fragments
                                and two N-glycan sites), a lack of templates and a two-residue sequence error <a
                                        href="https://www.nature.com/articles/313756a0">dating back to the original
                                    sequencing of the gene</a> used for the study caused substantial headaches for the
                                original crystallographers. On facing this for the very first time in 2013, one of my
                                first challenges was simply finding suitable visualisation modes to make it easier to
                                diagnose wide-ranging problems like this. Rebuilding this region makes use of
                                essentially all the tools available in ISOLDE 1.0b2, and can be achieved in about 20
                                minutes on a GPU workstation.</p>

                            <p>The 2018 cryo-EM structure of the insulin-bound receptor <a
                                    href="https://www.nature.com/articles/s41467-018-06826-6">reveals that this region
                                also appears likely to be important to insulin binding</a>, albeit in a somewhat
                                different manner to the FnIII-I domain. In the 3loh structure (with no insulin bound)
                                the two FnIII-3 domains of the dimer are far apart in space (forming the points of an
                                inverted "V" shape). Insulin binding triggers a dimerisation of these domains, with the
                                dimerisation surface consisting primarily of the region remodelled in this video.</p>

                        </div>


                    </div>


                </div>


            </div>
        </div>
    </div>
</section>


<div class="row">


    <div class="col col-1 col-sm-1 col-md-1 col-lg-1 col-xl-1">

    </div>


    <div class="col col-10 col-sm-10 col-md-10 col-lg-10 col-xl-10">

        <p>ISOLDE is developed and maintained by <a href="mailto:tcroll@altoslabs.com" class>Tristan Croll</a> at <a
                href="https://altoslabs.com">Altos Labs</a>. Development began in the <a
                href="https://www.cimr.cam.ac.uk/staff/professor-randy-j-read-frs" target="_blank" class>lab of Randy
            Read</a> at the <a href="https://www.cimr.cam.ac.uk/" target="_blank">Cambridge Institute for Medical
            Research</a>, supported by Prof. Read's Wellcome Trust Principal Research Fellowship 209407/Z/17/Z. Early
            development and testing was been aided by the generous donation of a GTX Titan Xp GPU by NVIDIA Corporation.
        </p>

    </div>


</div>


<div class="row">


    <div class="col col-1 col-sm-1 col-md-1 col-lg-1 col-xl-1">

    </div>


    <div class="col col-10 col-sm-10 col-md-10 col-lg-10 col-xl-10">

        <p>Website design by Tristan Croll based on <a href="https://templated.co/hielo" target="_blank">Hielo by
            Templated</a> under a CC BY 3.0 license.

            <a href="https://www.cam.ac.uk/about-this-site/privacy-policy">Privacy policy</a></p>

    </div>


</div>

<footer id="footer">
    <div class="container">
        <ul class="icons">
            <li>
                <a href="https://twitter.com/intent/tweet?text=ISOLDE%3A+bringing+structural+biology+to+life.+https://tristanic.github.io/isolde"
                   class="icon fa-twitter" target="_blank"><span class="label">Twitter</span></a></li>
            <li><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A//tristanic.github.io/isolde"
                   class="icon fa-facebook" target="_blank"><span class="label">Facebook</span></a></li>
            <li>
                <a href="https://www.linkedin.com/shareArticle?mini=true&url=https://tristanic.github.io/isolde&title=ISOLDE%3A%20Bringing%20structural%20biology%20to%20life.&source=ISOLDE"
                   class="icon fa-linkedin" target="_blank"><span class="label">LinkedIn</span></a></li>
            <li>
                <a href="mailto:?subject=ISOLDE for molecular model building&amp;body=Check out this cool model building package%3A https://tristanic.github.io/isolde."
                   class="icon fa-envelope-o"><span class="label">Email</span></a></li>
        </ul>
    </div>
</footer>

<!-- Hielo template code -->
<script src="../../static/isolde/assets/js/jquery.min.js"></script>
<script src="../../static/isolde/assets/js/jquery.scrollex.min.js"></script>
<script src="../../static/isolde/assets/js/skel.min.js"></script>
<script src="../../static/isolde/assets/js/util.js"></script>
<script src="../../static/isolde/assets/js/main.js"></script>
<script src="../../static/isolde/node_modules/jquery/dist/jquery.min.js"></script>
<script src="../../static/isolde/node_modules/bootstrap/dist/js/bootstrap.min.js"></script>


</body>
</html>
